### PR TITLE
Osgi libs

### PR DESCRIPTION
**Pull request by shroffk**
Cleaning up third party of unnecessary libs

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -181,54 +181,51 @@
...
-              <environment>
-                <os>macosx</os>
-                <ws>cocoa</ws>
-                <arch>x86</arch>
-              </environment>
+          <environments>
+            <environment>
+              <os>linux</os>
+              <ws>gtk</ws>
+              <arch>x86</arch>
```
Comment (berryma4) [inline pom.xml]: While we're at it, can we remove the non 64 bit targets?

Comment (shroffk) [inline pom.xml]: We still use 32 bit arch here at nsls2

Comment (kasemir) [inline pom.xml]: Same at SNS: For Linux, we need both 64 and 32 bit.
Windows and Mac are only 64.

Comment (berryma4) [inline pom.xml]: okay, ... I'll ask again in a year :+1:

Comment (kasemir) [inline pom.xml]: Unfortunately, the control system computers are closer to a 5 year update cycle, and just went to a new RedHat release - albeit still 32 bit - last year.

**Comment by berryma4** [comment]
@shroffk there seem to be conflicts.

**Pull request closed without merging**